src/main/java/entities/Postable.java

```diff
@@ -11,29 +9,22 @@
     User user;
     // the main text content of the Postable
     String body;
-    // a list of Postable objects created as a reply to this Postable
-    List<Postable> replies;
+
     // the date this Postable was created
-    Date creationDate;
+    String creationDate;
```

Comment:
I think the creationDate attribute should be left as a LocalDate, mainly so we can use the methods to compare datewise if needed.